app/assets/stylesheets/publisher.scss

```diff
@@ -243,14 +237,6 @@
     .hide-location { display: inline-block; }
     .locator { display: none; }
   }
-
-  .twitter-typeahead {
-    left: -1px;
-    // Override inline rule of Typeahead
-    // scss-lint:disable ImportantRule
-    position: absolute !important;
```

Comment:
> This only adds styles to the publisher (inside .publisher), so I don't see how that could break the comment mentions styles.

I see what you mean, I'll fix it now.

> The problem with the current version is the fact that mention suggestions are between the textarea and photos / location / polls / the submit button. This means that when you start mentioning someone everything below the suggestions might move with every single character you type.

Yes, and I don't see any simple solutions, since if I make it overlapping, it gets cut because of `overflow: hidden` property of the box. A possible solution could be to remove this property from the styles, but it requires to fix layout then, which is unobvious for me.